docs/12.features/7.attachments.md

```diff
@@ -14,6 +14,15 @@ Telegraph::message('hi')->withData('caption', 'test')->send();
 Telegraph::withData('caption', 'test')->message('hi')->send();
 ```
 
+## Custom Thread
+
+Attachments can be sent to a specific Thread through Telegraph `->thread()` method:
+
+```php
+Telegraph::message('hi')->thread('thread_id')->send();
+Telegraph::thread('thread_id')->message('hi')->send();
```

Comment:
I think we don't need this

```diff
Telegraph::message('hi')->thread('thread_id')->send();
-Telegraph::thread('thread_id')->message('hi')->send();
```

and we should also rename the method like this, for fluency

```suggestion
Telegraph::message('hi')->inThread('thread_id')->send();
```